bankholidays/src/main/java/com/celements/calendar/bankholidays/classes/BankHolidayClass.java

```diff
@@ -0,0 +1,54 @@
+package com.celements.calendar.bankholidays.classes;
+
+import java.util.Date;
+
+import javax.annotation.concurrent.Immutable;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+
+import com.celements.model.classes.AbstractClassDefinition;
+import com.celements.model.classes.fields.ClassField;
+import com.celements.model.classes.fields.DateField;
+import com.celements.model.classes.fields.StringField;
+
+@Singleton
+@Immutable
+@Component(BankHolidayClass.CLASS_DEF_HINT)
+public class BankHolidayClass extends AbstractClassDefinition implements BankHolidayClassesRole {
+
+  public static final String CLASS_NAME = "BankHolidayClass";
+  public static final String CLASS_SPACE = "Classes";
+  public static final String CLASS_DEF_HINT = CLASS_SPACE + "." + CLASS_NAME;
+
+  public static final ClassField<String> FIELD_HOLIDAY_NAME = new StringField.Builder(
```

Comment:
please add a validation regexp and dictionary entry to prevent too long names that would overflow the DB field